### ISSUE TITLE
Unknown reason for build failure

### ISSUE THREAD

**Issue by danielahedges** [Shippable/support#4082]
### Build fails in one branch, succeeds in another

Here are two builds in two different branches, of the same git version checksum:
- [125 - dev-branch succeeds](https://app.shippable.com/github/ePatientFinder/epf-app-simple/runs/125/summary/)
- [124 - new-account-service fails](https://app.shippable.com/github/ePatientFinder/epf-app-simple/runs/124/summary/)

I'm having trouble trying to peg down exactly what fails in the failure case. It appears all tests pass, all scripts checkboxes are green. I'm going through the logs, looking for what is different in these two cases. Do you have any input that could help?

**Comment by danielahedges** [comment]
It looks like a container is prepared, with the entrypoint being `/home/shippable/cexec/build.sh`, and in the failure case, that shell script exits with a code 99, whereas in the success case it exits with code 0.

I assume the Shippable engine is building this build.sh file, right? Assembling the pieces of my own build specifications? If so, is there a special meaning behind an exit code of 99?

**Comment by a-murphy** [comment]
It looks like your deployment script in `build_post_ci` might be returning a nonzero exit code.  I can't see the contents of that script, but you might want to check it.

If that doesn't help, we have seen a few cases in the past where a lot of logs printed very quickly can cause the build to exit suddenly, so you can also try reducing logging.

The `build.sh` file is assembled by Shippable to run the scripts in your `shippable.yml`, but 99 doesn't mean much more than that it failed.

**Comment by danielahedges** [comment]
Thanks for your response!

I looked in the contents of that script, and found two calls to "exit" without a return value. I changed those to an explicit "exit 0" to see if that would help. I strongly suspect they both were exiting with a zero, but meh.

Also, I changed the logging level of the previous step, `build_ci`, specifically inside the unit testing script, to see if that would help. It should have cut the size of the output about in half.

Apparently, one of those worked. My guess is it's the log level that did it.

Thanks for your help!
